examples/up.conf.example

```diff
@@ -24,13 +24,22 @@ quiet = false
 # an invalid mode will also result in "influx". In this default mode the application
 # runs as a daemon and polls the controller at the configured interval.
 #
-# There is only one other option at this time: "influxlambda"
+# There are two other options at this time: "influxlambda" and "prometheus"
```

Comment:
Shouldn't this be:
```suggestion
# There are two other options at this time: "influx" and "prometheus"
```

Reply:
Oh, nevermind, maybe change the line 29 below to say "influxlambda", so it's more clear.

Reply:
I updated the docs to clarify the modes.